assets/scss/optimized-checkout.scss

```diff
@@ -1 +1,111 @@
 // checkout scss file
+@import 'settings/settings';
+@import 'tools/tools';
+@import 'generic/generic';
+@import 'elements/elements';
+@import 'vendor/vendor';
+@import 'objects/objects';
+@import 'components/components';
+@import 'utilities/utilities';
+
+@mixin form-input() {
```

Comment:
Is it used in multiple places or just at line 40? If it's used in one place, you don't really need a mixin

Reply:
Done